### PR TITLE
Fix the GraphicsContext design issue in luminance-front.

### LINKED ISSUE #414: Some luminance-front types pass the crate::Backend to a position where a GraphicsContext is expected
This is a remnant of a _design bug_ where some types take a `B` where `C: GraphicsContext<Backend = B>` — i.e. impossible to know `C`, or not interesting, and some others take `C` directly.

Taking `C` directly only gives us the possibility to create new objects via the public interface of `GraphicsContext`, which is not really interesting, or to create pipeline gates.

### PR DESCRIPTION
**Pull request by phaazon**
Fixes #414.

### CODE REVIEW

**Comment by phaazon** [comment]
I’m not 100% sure how to release that fix though. It’s certain `luminance-front` must be `0.2`, but I don’t really want to release a breaking-change for that in `luminance`. I need to check whether a breaking-change is really mandatory here — I’m not quite sure.

File: luminance-front/src/render_gate.rs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+use crate::Backend;
+
+pub type Rendergate<'a> = luminance::render_gate::RenderGate<'a, Backend>;
```
Comment (m-hugo) [inline luminance-front/src/render_gate.rs:3]: why not RenderGate

Comment (phaazon) [reply luminance-front/src/render_gate.rs:3]: Typo. Thanks.

File: luminance-front/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "luminance-front"
-version = "0.1.0"
+version = "0.2.1"
```
Comment (phaazon) [inline luminance-front/Cargo.toml:3]: 0.2.0.

File: luminance-sdl2/CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
+
+# 0.1.1
+
+> Jul 21th, 2020
+
+- Support of `luminance-0.41`.
+
+# 0.1
+
+> Wed Jul 15th 2020
```
Comment (phaazon) [inline luminance-sdl2/CHANGELOG.md:27]: ,

File: luminance-web-sys/CHANGELOG.md [hunk 4]
```diff
@@ -11,10 +11,17 @@ compatible with as many crates as possible. In that case, you want `cargo update
...
 <!-- vim-markdown-toc GFM -->
 
+* [0.1.1](#011)
 * [0.1](#01)
 
 <!-- vim-markdown-toc -->
 
+# 0.1.1
+
+> Jul 21th, 2020
```
Comment (phaazon) [inline luminance-web-sys/CHANGELOG.md:21]: ,

File: luminance-web-sys/CHANGELOG.md [hunk 5]
```diff
@@ -11,10 +11,17 @@ compatible with as many crates as possible. In that case, you want `cargo update
...
+
+- Support of `luminance-0.41`.
+
 # 0.1
 
 > Wed Jul 15th 2020
```
Comment (phaazon) [inline luminance-web-sys/CHANGELOG.md:27]: ,

File: luminance-webgl/CHANGELOG.md [hunk 6]
```diff
@@ -11,10 +11,17 @@ compatible with as many crates as possible. In that case, you want `cargo update
...
 
+# 0.1.1
+
+> Jul 21th, 2020
+
+- Support of `luminance-0.41`.
+
 # 0.1
 
 > Wed Jul 15th 2020
```
Comment (phaazon) [inline luminance-webgl/CHANGELOG.md:27]: ,

**Pull request merged**